docs/painless/painless-contexts/painless-field-context.asciidoc

```diff
@@ -14,18 +14,15 @@ a customized value for each document in the results of a query.
         Contains the fields of the specified document where each field is a
         `List` of values.
 
-{ref}/mapping-source-field.html[`ctx['_source']`] (`Map`)::
+{ref}/mapping-source-field.html[`params['_source']`] (`Map`)::
```

Comment:
Do you mind adding read-only to this, like how _score was?  This shouldn't be modified as part of a script field.  My mistake from the first iteration of these docs.